tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/CI/CIEnvironmentVariableTests.cs

```diff
@@ -103,6 +103,20 @@ public void CheckEnvironmentVariables(JsonDataItem jsonData)
                         value = value.Replace(".000", string.Empty);
                     }
 
+                    if (spanDataItem.Key == CommonTags.CINodeLabels)
+                    {
+                        var labelsExpected = Datadog.Trace.Vendors.Newtonsoft.Json.JsonConvert.DeserializeObject<string[]>(
+                            spanDataItem.Value);
+                        Array.Sort(labelsExpected);
+
+                        var labelsActual =
+                            Datadog.Trace.Vendors.Newtonsoft.Json.JsonConvert.DeserializeObject<string[]>(value);
+                        Array.Sort(labelsActual);
+
+                        Assert.True(labelsExpected.SequenceEqual(labelsActual));
```

Comment:
FluentAssertions gives nicer error messages :p
```suggestion
                        labelsActual.Should().Equal(labelsExpected);
```

Reply:
done in [df3a6a6](https://github.com/DataDog/dd-trace-dotnet/pull/4049/commits/df3a6a68b29d2d548bd006e1204e9c793b87c18e)